Example/Tests/BaseNetwork/BaseTestCase.swift

```diff
@@ -26,7 +26,7 @@ import Foundation
 import XCTest
 
 class BaseTestCase: XCTestCase {
-    let timeout: TimeInterval = 30.0
+    let timeout: TimeInterval = 40.0
```

Comment:
Please make this number consistent with the one from the Obj-c test cases (perhaps use a the same const in both places).